Emby.Server.Implementations/HttpServer/WebSocketConnection.cs

```diff
@@ -221,12 +219,6 @@ private async Task ProcessInternal(PipeReader reader)
             };
 
             await OnReceive(info).ConfigureAwait(false);
-
-            // Stop reading if there's no more data coming
-            if (result.IsCompleted)
-            {
-                return;
-            }
```

Comment:
why remove?

Reply:
It's the end of the function, it would return anyway?

Reply:
LOL, I thought it was doing stuff in a loop